databases/week3/queries.sql

```diff
@@ -0,0 +1,69 @@
+SELECT *
+FROM `meal`;
```

Comment:
Why parenthesis around table name when quering ...? Remove all parenthesis where you use it around table name
for instance in lines. 2,6.8,12,15 etc